website/app/components/doc/icons-list/grid.hbs

```diff
@@ -3,13 +3,9 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
+<h2 class="doc-text-h2">{{capitalize @category}}</h2>
```

Comment:
@KristinLBradley The [category title](https://www.figma.com/file/Ky0qWjvHZR3je1lCBlzNB1/HDS-website?type=design&node-id=4462%3A240458&mode=design&t=rPxzjDyrvG0ZZpwo-1) is styled as a Header-4, but would it be possible to respect the heading order and apply a different visual order? For example, a `<h2>` with a `heading-4` style here?

Reply:
Sure, thanks I missed that. I will update.